### ISSUE TITLE
Unable to install eugenmayer/dockersync/unison

### ISSUE THREAD

**Issue by localheinz** [EugenMayer/homebrew-dockersync#10]
### Steps required to reproduce

1. Run

    ```
    $ brew tap eugenmayer/dockersync
    ```

2. Run

    ```
    $ brew install eugenmayer/dockersync/unison
    ```

### Expected Result

Installation succeeds.

### Actual Result

Installation fails.

```
==> Installing unison from eugenmayer/dockersync
==> Downloading https://homebrew.bintray.com/bottles-dockersync/unison-2.51.2.high_sierra.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "unison"
Download failed: https://homebrew.bintray.com/bottles-dockersync/unison-2.51.2.high_sierra.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/bcpierce00/unison/archive/v2.51.2.tar.gz
Already downloaded: /Users/am/Library/Caches/Homebrew/downloads/558b42a95b847ce902780f27b7bc703d8c70b855f07b629a826abc807b99949c--unison-2.51.2.tar.gz
==> make src/mkProjectInfo
Last 15 lines from /Users/am/Library/Logs/Homebrew/unison/01.make:
2019-02-04 13:30:42 +0100

make
src/mkProjectInfo

make: *** No rule to make target `src/mkProjectInfo'.  Stop.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/eugenmayer/homebrew-dockersync/issues
```

### Additional Information

Running

```
$ sw_vers
```

yields

```
ProductName:	Mac OS X
ProductVersion:	10.14.3
BuildVersion:	18D42
```

and running

```
$ brew --version
```

yields

```
Homebrew 2.0.0
Homebrew/homebrew-core (git revision 385c9; last commit 2019-02-04)
Homebrew/homebrew-cask (git revision 8ab02; last commit 2019-02-03)
```
💁‍♂️ See [https://homebrew.bintray.com/bottles-dockersync/unison-2.51.2.high_sierra.bottle.tar.gz](https://homebrew.bintray.com/bottles-dockersync/unison-2.51.2.high_sierra.bottle.tar.gz).

**Comment by djsonbim** [comment]
I'm install eugenmayer/dockersync/unox but it not install for me :(
Help me, and thanks for help!

Error: 
`Last 15 lines from /Users/hoangson/Library/Logs/Homebrew/unox/01.python2:
-c
import setuptools, tokenize
__file__ = 'setup.py'
exec(compile(getattr(tokenize, 'open', open)(__file__).read()
  .replace('\r\n', '\n'), __file__, 'exec'))
--no-user-cfg
install
--prefix=/private/tmp/unox--homebrew-virtualenv-20190211-16458-1wodq4d/target
--install-scripts=/private/tmp/unox--homebrew-virtualenv-20190211-16458-1wodq4d/target/bin
--single-version-externally-managed
--record=installed.txt

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named setuptools

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/eugenmayer/homebrew-dockersync/issues`

**Comment by reetz250** [comment]
I had a similar problem, I was able to get around it by uninstalling python
`brew uninstall python@2`

Hope that helps

**Comment by azharkhan** [comment]
still having this same problem :( even tried the uninstall but no further luck. 

I'm on macOS 10.14.5 (Mojave)

**Comment by EugenMayer** [comment]
I will No loner support this tap, please either install it manually or apply for being contributor/maintainer and try to fix this - please do not wait for a fix on my side, this would be a waste of time :)

**Comment by EugenMayer** [comment]
should be fixed by #11
